pkg/dispatcher/testdata/TestLoadConfig/good_config_with_build_farm_with_jobs.yaml

```diff
@@ -1,5 +1,5 @@
 default: api.ci
-nonKubernetes: app.ci
+nonKubernetes: ""
```

Comment:
I think you can just remove this field, it doesn't serve any purpose

Reply:
The purpose is that we have this field (not its value) `nonKubernetes` defined/implemented?

Reply:
This field must be somewhere in the go type, but is completely irrelevant what cluster non-Kubernetes type jobs are on (could probably be hardcoded to the empty string)

Reply:
Removed.